docs/reference/ilm/actions/ilm-shrink.asciidoc

```diff
@@ -4,7 +4,7 @@
 
 Phases allowed: hot, warm.
 
-Sets a source index to <<index-blocks-read-only,read-only>> and shrinks it into
+<<index-blocks-write,Blocks writes>> on a source index and shrinks it into
```

Comment:
The phrase "Blocks writes on a source index ..." is not as clear as "Sets a source index to read-only...", but since `index.blocks.write` is what actually gets set, rather than `index.blocks.read-only`, I think the current phrasing confusing.